### PR TITLE
[MRG] udpate doc of mne_anonymize

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: doc/manual/c_reference.rst [hunk 1]
```diff
@@ -3189,12 +3189,13 @@ file:
     FIFF_SUBJ_WEIGHT          Weight of the subject in kg
     FIFF_SUBJ_HEIGHT          Height of the subject in m
     FIFF_SUBJ_COMMENT         Comment about the subject
-    ========================  ==============================================
+    FIFF_MEAS_DATE            The date of the recordings
```
Comment (agramfort) [inline doc/manual/c_reference.rst]: this option was added in March 2016 by @mshamalainen

Comment (agramfort) [reply doc/manual/c_reference.rst]: in PROGRAM_VERSION     "1.4"

**Comment by agramfort** [comment]
looks good : https://7103-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.sC9zP29/html/manual/c_reference.html?highlight=mne_analyze#mne-anonymize

good to go from my end

**Comment by larsoner** [comment]
Did you comment on the PR just to have a record somewhere or did you mean to add those to the doc?

**Comment by agramfort** [comment]
we don't document version chagnes in the C doc so it was just to tell you :)

**Comment by larsoner** [comment]
Thanks @agramfort

**Pull request merged**